ngraph/frontend/ir/include/ir_frontend/model.hpp

```diff
@@ -15,18 +15,15 @@ namespace ngraph {
 namespace frontend {
 class IR_API InputModelIR : public InputModel {
     friend class FrontEndIR;
-
+    std::shared_ptr<ngraph::op::Constant> m_weights;
+    ov::Extensions m_extensions;
     pugi::xml_node m_root;
-    InferenceEngine::Blob::CPtr m_weights;
-    std::vector<InferenceEngine::IExtensionPtr> m_exts;
+    pugi::xml_document m_xml_doc;
```

Comment:
we need to hide pugixml from public API. Probably, https://en.cppreference.com/w/cpp/language/pimpl could help

Reply:
I used forward declaration and pointers to xml_node and xml_document objects.